README.md

```diff
@@ -1,5 +1,54 @@
-# 2dTo3dVector
-## Additions
-A CLI to convert 2D vector data to 3D vector data. 
-Testing a commit with VSCode
-Now, submitting a Pull Request instead of a Commit
\ No newline at end of file
+# Mountaineer
+A CLI to convert 2D vector data to 3D vector data.  
+
+## Usage
+CLI Usage: One liner
+
+mountaineer [input file] [lat] [lon] [output file]
```

Comment:
You should add a command name here, incase you want to add more commands down the road, and then you won't be making a breaking change. For example:
`mountaineer get-elevations ([options]) [input-file] [lat] [lon] [output-path]`

Reply:
Command added and parameters updated.